tests/optimizer/applyImportAndExportNameChanges-output.js

```diff
@@ -0,0 +1,21 @@
+var name;
+Module.asmLibraryArg = {
+ "a": 1,
+ "A": 33,
+ "b": ___syscall6,
+ "__setErrNo": ___setErrNo,
+ "memset": _memset,
+ "sbrk": _sbrk,
+ "memcpy": _memcpy,
+ "emscripten_memcpy_big": _emscripten_memcpy_big,
+ "c": ___syscall54,
+ "d": ___syscall140,
+ "a": ___syscall146
```

Comment:
Is there meant to be two `a` keys here? I see that's in the mapping below, but it would probably be clearer to have non-overwritten keys even just for the test?

Reply:
Thanks @curiousdannii, that wasn't intentional, I combined two testcases improperly. Fixing.